package.json

```diff
@@ -30,6 +30,11 @@
     "canjs-plugin",
     "donejs"
   ],
+  "steal": {
+    "plugins": [
+      "steal-css"
```

Comment:
do I need to add `steal-qunit` to the plugins config? It doesn't seem to work otherwise. I didn't see that in the docs anywhere so want to make sure.